src/Commands/SetBrake.cpp

```diff
@@ -0,0 +1,41 @@
+#include "SetBrake.h"
+#include "../Robot.h"
+
+SetBrake::SetBrake(Chassis::TalonMode mode)
+: mMode(mode)
+{
+  // Use Requires() here to declare subsystem dependencies
+  // eg.
+  Requires(Robot::chassis.get());
+}
+
+// Called just before this Command runs the first time
+void SetBrake::Initialize()
+{
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void SetBrake::Execute()
+{
+  Robot::chassis->SetMode(mMode);
```

Comment:
SetMode is a one time function, initialize could be a better place for it. There is no issue here as long as the command doesn't repeat